DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: Rcpp
 Title: Seamless R and C++ Integration
-Version: 1.0.6.4
-Date: 2021-01-21
+Version: 1.0.6.5
+Date: 2021-01-28
```

Comment:
It's probably best to let me (release manager) handle that though that is what I did locally to distinguish the rev dep run.